specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,58 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all charges with the same [**SKU Price ID**](#skupriceid). These properties enable practitioners to calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+* The properties (both keys and values) contained in the _SkuPriceDetails_ column MUST be shared across all charges having the same _SkuPriceId_.
+  * Additional properties (key-value pairs) MAY be added to _SkuPriceDetails_ for a _SkuPriceId_.
+  * Properties SHOULD NOT be removed from _SkuPriceDetails_ for a _SkuPriceId_, once they have been included.
+  * Individual properties (key-value pairs) SHOULD NOT be modified for a _SkuPriceId_.
```

Comment:
Individual properties (key-value pairs) SHOULD NOT be modified for a given SkuPriceId. They should be the same every time they appear for that SkuPriceId.

Reply:
Took another go at this to incorporate your feedback without breaking into 2 sentences.
"Individual properties (key-value pairs) SHOULD NOT be modified for a _SkuPriceId_ and SHOULD remain consistent over time."

The other aspect of your statement is covered in row 10, which this is a sub-bullet under.

Reply:
Also added "given" to each of these 3 sub-bullets.